README.md

```diff
@@ -4,6 +4,19 @@ Proxy that provides access to the VNC console of a Kubevirt VM.
 
 It can generate time limited tokens that are then used to access VNC.
 
+## Installation
+
+### Using SSP operator
+The [SSP operator](https://github.com/kubevirt/ssp-operator) can be configured to install VM Console Proxy together with
+a Route used to expose it to the outside.
+
+### Without SSP operator
+The VM Console Proxy needs to be deployed in the same namespace where Kubevirt is.
```

Comment:
I'd write:
`
VM Console Proxy needs to be deployed in the same namespace as KubeVirt.
To deploy the latest version, use the following command:
...
`

Reply:
done